azure-iot-device/azure/iot/device/iothub/transport/pipeline_ops_iothub.py

```diff
@@ -97,3 +97,26 @@ def __init__(self, message, callback=None):
         super(SendOutputEvent, self).__init__(callback=callback)
         self.message = message
         self.needs_connection = True
+
+
+class SendMethodResponse(PipelineOperation):
+    """
```

Comment:
>   """ [](start = 1, length = 6)

odd indents in this function

Reply:
Good catch. My indenting got automatically messed up due to the double spaces you added on continued docstring lines elsewhere in the file. I believe the MS docgen convention is to only use one space for that. I'll fix this, and also the other docs that caused it.

---
In reply to: [282743332](https://github.com/Azure/azure-iot-sdk-python-preview/pull/86#discussion_r282743332) [](ancestors = 282743332)